specs/language/statements.tex

```diff
@@ -1,2 +1,93 @@
 \Ch{Statements}{Stmt}
 \Sec{Label Statements}{Stmt.Label}
+\Sec{Attributes}{Stmt.Attr}
+\Sub{Unroll Attribute}{Stmt.Attr.Unroll}
+The \texttt{[unroll]} and \texttt{[unroll(n)]} attribute qualifiers can be used
+ to specify that a loop (\texttt{for}, \texttt{while} and \texttt{do while} 
+loops) can be unrolled. This attribute qualifier can be used to specify full 
+unrolling or partial unrolling by a specified amount. This is a compiler hint 
+and the compiler may ignore this directive.
+
+\texttt{n} is the loop unrolling factor and must be a positive integral compile time 
```

Comment:
The wording of this might be more clear as something like:
> The _unroll_ attribute may optionally have an a single argument that is an integer constant expression value greater than zero.